File: baekjoon/set/BOJ1269.cpp
#include <iostream>
#include <algorithm>
#include <vector>

using namespace std;

int main()
{
  ios::sync_with_stdio(false);
  cin.tie(NULL);
  cout.tie(NULL);

  int n, m;
  cin >> n >> m;

  vector<int> arr(n);
  for (int i = 0; i < n; i++)
  {
    cin >> arr[i];
  }

  sort(arr.begin(), arr.end());

  int cnt = 0;
  for (int i = 0; i < m; i++)
  {
    int temp;
    cin >> temp;
    if (binary_search(arr.begin(), arr.end(), temp) == 1)
    {
      cnt++;
    }
  }

  cout << n + m - 2 * cnt;
}

File: baekjoon/set/BOJ1269.java
package baekjoon.set;

import java.io.*;
import java.util.*;

public class BOJ1269 {

    public static void main(String[] args) throws IOException {
        int N, M = 0;
        ArrayList<Integer> arr;
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String[] split = br.readLine().split(" ");
            N = Integer.parseInt(split[0]);
            M = Integer.parseInt(split[1]);

            arr = new ArrayList<>(N);
            split = br.readLine().split(" ");
            for (int i = 0; i < N; i++) {
                arr.add(Integer.parseInt(split[i]));
            }

            Collections.sort(arr);

            split = br.readLine().split(" ");
            int cnt = 0;
            for (int i = 0; i < M; i++) {
                if (Collections.binarySearch(arr, Integer.parseInt(split[i])) >= 0)
                    cnt++;
            }

            System.out.print(N + M - cnt * 2);
        }
    }
}
